jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -471,6 +471,56 @@
               echo "Exiting with code: ${{rc}}"
               exit ${{rc}}
             fi
+      - build-test-kubemark-e2e-gce: # kubernetes-pull-build-test-kubemark-e2e-gce
+          status-context: Kubemark GCE e2e
+          max-total: 12
+          trigger-phrase: 'kubemark\s+test'
```

Comment:
as currently configured, it will trigger for all open PRs. use of the "purge build queue" button a few times may be justified...